src/main/resources/application.properties

```diff
@@ -11,3 +11,4 @@ spring.h2.console.enabled=true
 spring.h2.console.path=/h2-console
 spring.h2.console.settings.trace=false
 spring.h2.console.settings.web-allow-others=false
+spring.profiles.active=local
```

Comment:
cannot change profile by changing this property
and i think this is old way 
we have profile in IntelliJ IDEA(right in run section) and its better to set there 
feel free to change configuration and switch it to `yml` format instead of  `properties` file
![active-profile](https://user-images.githubusercontent.com/58255161/216788419-1c250910-efe1-4f8d-9b41-87374c0fae3b.png)
@DanialMehrabi94

Reply:
i changed the configuration to yaml 
special tnx to you mohammad reza i didn't know about that
but intellij still doesn't suggest me active profiles on run configuration
i cloned some other spring boot projects with yaml configuration and all of them didn't suggest active profile